### ISSUE TITLE
Unknown option "--fixIncompleteTypes"

### ISSUE THREAD

**Issue by Coteh** [JoshuaKGoldberg/TypeStat#785]
### 🐛 Bug Report

-   TypeStat version: 0.4.2
-   TypeScript version: 4.0.3
-   Node version: 12.18.4

Hi, I recently heard of this project and was looking to see if something like this can be used to migrate a JavaScript project over to TypeScript.

I ran the sample project with a trivial .js file added into it to see if it can rename js files to ts and add type annotations but I'm presented with this error message:

```
The following JavaScript files were included in the project but --fileRenameExtensions is not enabled.
TypeStat does not yet support annotating JavaScript files.
```

I add the flag to `typestat` run command then I get `error: unknown option '--fileRenameExtensions'`.

#### Actual Behavior

`error: unknown option '--fileRenameExtensions'`

#### Expected Behavior

.js file gets renamed to .ts and gets TypeScript typings

#### Reproduction

1. Checkout sample project https://github.com/JoshuaKGoldberg/typestat-example
2. Create a file called `example.js` inside `src` with following content:
```
function doSomething(someParam) {
    console.log(someParam);
}

doSomething("hello world");
```
3. Change `include` value in `typestat.json` to `["src/*"]`
4. Change `typestat` version in `package.json` dependencies to `^0.4.2` and run `npm install`
5. Run `typestat --config ./typestat.json --fileRenameExtensions`

If I rename the file manually, then run it without the flag, I get the type annotations as expected:

```
function doSomething(someParam: string) {
    console.log(someParam);
}

doSomething("hello world");
```

But it'd be nice if we could also have all js files changed to ts as well in one step.

I skimmed some of the previously posted issues in this repo and it seems like you've added some test cases already for `--fileRenameExtensions` so it seems like there must be some fluke and that it should be working.

Let me know if I misunderstood anything about this project or if this is something that can be fixed. Thanks.

edit: also noticed this is the output of `--help`. Looks like `--fileRenameExtensions` is not even mentioned as an option: 🤔 

```
Usage: typestat [options]

Options:
  -c --config [config]    path to a TypeStat config file
  -l --logfile [logfile]  paths to a logfile to
  -V --version            output the package version
  -h, --help              display help for command
```

**Comment by JoshuaKGoldberg** [comment]
:wave: @Coteh, thanks for trying this out! That message you got at first about `--fixIncompleteTypes`/`--fileRenameExtensions` is a relic of back when TypeStat could take a lot more options on the CLI. It's since changed to only allowing them in the `--config` file (e.g. `typestat.json`), and it's a CLI/documentation bug that the `--` flags are still referenced. 

Accepting PRs to fix the bug - if you / nobody else sends one by this weekend, I should be able to. Thanks for reporting! 🙌 

As for what to do instead, you'll want a config file that includes all the fixers and settings you need. Until the documentation has more samples, here's an example of what I think you might want:

```json
{
  "files": {
    "renameExtensions": true
  },
  "fixes": {
    "incompleteTypes": true,
    "noImplicitAny": true
  },
  "include": ["src/index.ts"],
  "projectPath": "./tsconfig.json"
}
```

**Comment by JoshuaKGoldberg** [comment]
Oh you know what, @Coteh this was actually fixed a bit back: #243 -> #677. The CLI version you're running with must be an older one.

```
The following JavaScript files were included in the project but files.renameExtensions is not enabled.
TypeStat does not yet support annotating JavaScript files.
...
See https://github.com/JoshuaKGoldberg/TypeStat/blob/main/docs/Files.md for details.
```

Version 0.4.1 is the latest right now and includes the above fix, along with a bunch of others. I'd recommend using that.
